conferences/tests.py

```diff
@@ -9,9 +9,18 @@
 from users.models import Organization
 from .models import Zosia, Place, UserPreferences, Bus
 from .forms import UserPreferencesForm
-from .test_helpers import new_zosia, new_user, new_bus, PRICE_ACCOMODATION, PRICE_BASE, \
-    PRICE_BONUS, PRICE_BREAKFAST, PRICE_DINNER, PRICE_TRANSPORT
-
+from .test_helpers import (
+    new_zosia,
+    new_user,
+    new_bus,
+    user_preferences,
+    PRICE_ACCOMODATION,
+    PRICE_BASE,
+    PRICE_BONUS,
+    PRICE_BREAKFAST,
+    PRICE_DINNER,
+    PRICE_TRANSPORT
+)
```

Comment:
if you want to make import with that style then maybe use `from . import test_helpers`? Too many lines wasted for imports.